aws/build.rake

```diff
@@ -272,51 +293,49 @@ end
 
 task :pegasus_unit_tests do
   Dir.chdir(pegasus_dir) do
-    HipChat.log 'Running <b>pegasus</b> unit tests...'
-    begin
+    with_hipchat_logging("pegasus unit tests") do
       RakeUtils.rake 'test'
-    rescue
-      HipChat.log 'Unit tests for <b>pegasus</b> failed.', color: 'red'
-      HipChat.developers 'Unit tests for <b>pegasus</b> failed.', color: 'red', notify: 1
-      raise
     end
   end
 end
 
 task :shared_unit_tests do
   Dir.chdir(shared_dir) do
-    HipChat.log 'Running <b>shared</b> unit tests...'
-    begin
+    with_hipchat_logging("shared unit tests") do
       RakeUtils.rake 'test'
-    rescue
-      HipChat.log 'Unit tests for <b>shared</b> failed.', color: 'red'
-      HipChat.developers 'Unit tests for <b>shared</b> failed.', color: 'red', notify: 1
-      raise
     end
   end
 end
 
-task :dashboard_unit_tests do
+def log_coverage_results
+  results_file = dashboard_dir('coverage/.last_run.json')
+  results = JSON.parse(File.read(results_file))
+  HipChat.log "Unit tests for <b>dashboard</b> coverage: #{results["result"]["covered_percent"]}%. Details: https:#{CDO.studio_url('coverage/index.html')}", color: 'green'
```

Comment:
do you think it makes sense to change all of the messages to be tagged "ruby unit tests" now that we also have js unit tests for dashboard?